services/packagist/packagist-version.spec.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+// const { expect } = require('chai')
+// const PackagistVersion = require('./packagist-version.service')
+
+describe('PackagistVersion', function() {})
```

Comment:
This is where I was starting to add tests to cover some of the transform logic, but decided to hold off on that for now (see my comment in PR description).